src/Components/Complaint/Complaint.js

```diff
@@ -12,7 +12,7 @@ const Complaint = () => {
     console.log("complaint:", complaint);
 
     setComplaint("");
-    setComplaint("");
+    setComplaintSubject("");
```

Comment:
After submitiing form the fields are not becoming empty, can we fix this or we can also do that after submitting the form the fields becomes uneditable.